x-pack/plugins/fleet/server/services/fleet_server/elastic_index.ts

```diff
@@ -0,0 +1,114 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { ElasticsearchClient } from 'kibana/server';
+import hash from 'object-hash';
+
+import { FLEET_SERVER_INDICES } from '../../../common';
+import { appContextService } from '../app_context';
+import ESFleetAgentIndex from './elasticsearch/fleet_agents.json';
+import ESFleetPoliciesIndex from './elasticsearch/fleet_policies.json';
+import ESFleetPoliciesLeaderIndex from './elasticsearch/fleet_policies_leader.json';
+import ESFleetServersIndex from './elasticsearch/fleet_servers.json';
+import ESFleetEnrollmentApiKeysIndex from './elasticsearch/fleet_enrollment_api_keys.json';
+import EsFleetActionsIndex from './elasticsearch/fleet_actions.json';
+
+const INDEX_VERSION = 1;
```

Comment:
is the idea that we would update this whenever we make a mapping change to any fleet index?

if so, maybe this should be named `FLEET_SERVER_INDICES_VERSION` and exported from the same place as `FLEET_SERVER_INDICES`?

Reply:
Yes the idea if we do a destructive migration one day we could upgrade this number